src/libs/actions/Policy.ts

```diff
@@ -19,8 +19,41 @@ import * as ReportUtils from '@libs/ReportUtils';
 import * as TransactionUtils from '@libs/TransactionUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {PersonalDetailsList, Policy, PolicyMember, PolicyTags, RecentlyUsedCategories, RecentlyUsedTags, ReimbursementAccount, Report, ReportAction, Transaction} from '@src/types/onyx';
+import type {Errors} from '@src/types/onyx/OnyxCommon';
+import type {CustomUnit} from '@src/types/onyx/Policy';
+import {isEmptyObject, isNotEmptyObject} from '@src/types/utils/EmptyObject';
```

Comment:
@cdOut Let's remove `isNotEmptyObject` in favor of `isEmptyObject` (is being removed [here](https://github.com/Expensify/App/pull/34490))